src/game/client/gameclient.cpp

```diff
@@ -557,12 +566,30 @@ void CGameClient::OnMessage(int MsgId, CUnpacker *pUnpacker)
 				m_pSounds->Enqueue(CSounds::CHN_GLOBAL, SOUND_CTF_RETURN);
 				break;
 			case GAMEMSG_TEAM_ALL:
-				str_format(aBuf, sizeof(aBuf), Localize("All players were moved to the %s"), GetTeamName(aParaI[0], m_GameInfo.m_GameFlags&GAMEFLAG_TEAMS));
-				m_pBroadcast->DoBroadcast(aBuf);
+				{
+					const char *pMsg;
+					switch(GetStrTeam(aParaI[0], TeamPlay))
+					{
+					case STR_TEAM_GAME: pMsg = Localize("All players were moved to the game"); break;
+					case STR_TEAM_RED: pMsg = Localize("All players were moved to the red team"); break;
+					case STR_TEAM_BLUE: pMsg = Localize("All players were moved to the blue team"); break;
+					case STR_TEAM_SPECTATORS: pMsg = Localize("All players were moved to the spectators"); break;
+					}
+					m_pBroadcast->DoBroadcast(pMsg);
+				}
 				break;
 			case GAMEMSG_TEAM_BALANCE_VICTIM:
-				str_format(aBuf, sizeof(aBuf), Localize("You were moved to %s due to team balancing"), GetTeamName(aParaI[0], m_GameInfo.m_GameFlags&GAMEFLAG_TEAMS));
-				m_pBroadcast->DoBroadcast(aBuf);
+				{
+					const char *pMsg;
+					switch(GetStrTeam(aParaI[0], TeamPlay))
+					{
+					case STR_TEAM_GAME: pMsg = Localize("You were moved to game due to team balancing"); break;
+					case STR_TEAM_RED: pMsg = Localize("You were moved to red due to team balancing"); break;
+					case STR_TEAM_BLUE: pMsg = Localize("You were moved to blue due to team balancing"); break;
+					case STR_TEAM_SPECTATORS: pMsg = Localize("You were moved to spectators due to team balancing"); break;
```

Comment:
There are missing `the` in all 4 strings

Reply:
Also missing the word "team" after red/blue